lib/help_scout.rb

```diff
@@ -229,23 +230,37 @@ def search(path, query, page_id = 1, items = [])
     end
   end
 
+  def get_token
+    options = {
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: {
+        grant_type: "client_credentials",
+        client_id: @api_key,
+        client_secret: @api_secret
+      }
+    }
+    options[:body] = options[:body].to_json
+    response = HTTParty.send(:post, 'https://api.helpscout.net/v2/oauth2/token', options)
+    token = JSON.parse(response.body)['access_token']
+    "Bearer #{token}"
+  end
+
   def request(method, path, options)
-    uri = URI("https://api.helpscout.net/v1/#{path}.json")
+    uri = URI("https://api.helpscout.net/v2/#{path}")
 
     # The password can be anything, it's not used, see:
     # http://developer.helpscout.net/help-desk-api/
```

Comment:
@gurs1kh might want to also remove this comment ✂️

Reply:
Thanks! I've fixed it in the latest commit